presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -96,6 +96,125 @@ Finally, you can access the ``clicks`` table in the ``web`` database::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``mysql`` in the above examples.
 
+Type mapping
+------------
+
+Because PrestoDB and MySQL each support types that the other does not. When reading from or writing to MySQL,
```

Comment:
```suggestion
PrestoDB and MySQL each support types that the other does not. When reading from or writing to MySQL,
```
"Because" is not needed at the beginning of this sentence. If this sentence was the beginning of a longer sentence it would be appropriate here. "PrestoDB and MySQL each support types that the other does not." states the problem, and the rest of this section explains how the problem is addressed.